setup.cfg

```diff
@@ -59,3 +48,21 @@ exclude =
     manage.py,
     .venv
 max-complexity = 16
+
+[mypy]
```

Comment:
you can move this to pyproject.toml

Reply:
good point, I'll do that in a follow-up PR